src/monodroid/jni/monodroid-glue.cc

```diff
@@ -860,57 +886,6 @@ MonodroidRuntime::LocalRefsAreIndirect (JNIEnv *env, jclass runtimeClass, int ve
 	return 1;
 }
 
-int
```

Comment:
Any reason to remove this member function in *this* PR and not a separate one?

Reply:
I don't think it warrants a full PR to remove a single function... Such a PR would bring nothing but a waste of time IMO